src/core/scene/a-scene.js

```diff
@@ -45,13 +45,14 @@ module.exports = registerElement('a-scene', {
 
     createdCallback: {
       value: function () {
-        this.isMobile = isMobile;
+        this.init();
         this.isIOS = isIOS;
+        this.isMobile = isMobile;
         this.isScene = true;
         this.object3D = new THREE.Scene();
+        this.renderBinded = this.render.bind(this);
```

Comment:
I get `a-scene.js:405 Uncaught TypeError: Cannot read property 'render' of undefined`